setup.cfg

```diff
@@ -106,7 +106,7 @@ apple =
     thinc-apple-ops>=0.0.4,<1.0.0
 # Language tokenizers with external dependencies
 ja =
-    sudachipy>=0.4.9
+    sudachipy>=0.6.2
```

Comment:
Because of the compatibility/compiling issues, how about `>=0.4.9,!=0.6.1` instead?

The versions used for the models can be managed separately, if necessary.

Reply:
On the other hand the extra barely matters, so it's probably fine.

Reply:
Yeah, I wouldn't be worried about accidentally installing 6.1, but I think when we bump things like this we might be inclined to use 6.0 as a base here normally, and I wanted to avoid that. 

If we want to keep the lower base and just not do this that's fine too, but the speed bump is probably worth it for most users, so if it's not disruptive to update our models I would suggest we go ahead with it.

Since wheel support is pretty good, I would expect that most users wouldn't even notice the fact that it's using Rust.

Reply:
Well, there aren't any arm64 wheels and having to have rust makes the source install a lot trickier for your average user.

Reply:
I think if the old version still works I would leave it here.

Reply:
Oh, I wasn't aware of the lack of arm64 support. In that case I guess we can leave the old version.

Reply:
To follow up again, I'd be fine with either leaving it as-is or using `>=0.4.9,!=0.6.1` to make this problem explicit. It probably would be a good idea to make it explicit in the trained pipelines either way.

Reply:
OK, let's change to the explicit version spec so we don't forget it. I'll update this PR.